.github/workflows/release.yml

```diff
@@ -38,6 +38,7 @@ jobs:
       - name: Fetch master branch
         run: |
           git fetch --depth=1 origin master:master
+          git checkout master
```

Comment:
Apparently we simply need to actually checkout master to avoid that error about the PR base being invalid.

Reply:
I think this step should be just before the "Bump patch version" step, since we want to build and release the extension based on the commit with the tag, whereas we want the version bump PR to be based on `master`.

Reply:
Good call - those are extensionally the same as I've been doing the release, but better to match intension as well.